test/e2e/node/kubelet.go

```diff
@@ -310,52 +302,73 @@ var _ = SIGDescribe("kubelet", func() {
 			}
 		})
 
-		for _, itArg := range deleteTests {
-			name := fmt.Sprintf(
-				"kubelet should be able to delete %d pods per node in %v.", itArg.podsPerNode, itArg.timeout)
-			ginkgo.It(name, func() {
-				totalPods := itArg.podsPerNode * numNodes
-				ginkgo.By(fmt.Sprintf("Creating a RC of %d pods and wait until all pods of this RC are running", totalPods))
-				rcName := fmt.Sprintf("cleanup%d-%s", totalPods, string(uuid.NewUUID()))
-
-				err := framework.RunRC(testutils.RCConfig{
-					Client:       f.ClientSet,
-					Name:         rcName,
-					Namespace:    f.Namespace.Name,
-					Image:        imageutils.GetPauseImageName(),
-					Replicas:     totalPods,
-					NodeSelector: nodeLabels,
-				})
-				framework.ExpectNoError(err)
-				// Perform a sanity check so that we know all desired pods are
-				// running on the nodes according to kubelet. The timeout is set to
-				// only 30 seconds here because framework.RunRC already waited for all pods to
-				// transition to the running status.
-				err = waitTillNPodsRunningOnNodes(f.ClientSet, nodeNames, rcName, ns, totalPods, time.Second*30)
-				framework.ExpectNoError(err)
-				if resourceMonitor != nil {
-					resourceMonitor.LogLatest()
-				}
+		ginkgo.It("kubelet should be able to delete %d pods per node in %v", func() {
```

Comment:
Very possible I'm missing something... but what is filing in the template values in this string? I think we may need a `fmt.Sprintf`.

Reply:
All test names must be literal, although this contains the templates it's still a literally string.
Perhaps a new name is needed?

Reply:
@BobyMCbobs and I discussed this and realized that there was a misunderstanding of the request in this [comment](https://github.com/kubernetes/kubernetes/pull/80731#issuecomment-524410321). Our understanding now is that this test should remain **unchanged**, and this PR will add the new test below with the purpose of meeting Conformance requirements.